README.md

```diff
@@ -415,41 +415,49 @@ The function `cml-runner` accepts the following arguments:
 Usage: cml-runner.js
 
 Options:
-  --version               Show version number                          [boolean]
-  --labels                Comma delimited runner labels         [default: "cml"]
-  --idle-timeout          Time in seconds for the runner to be waiting for jobs
-                          before shutting down. 0 waits forever.  [default: 300]
-  --name                  Name displayed in the repo once registered
-                                                     [default: "cml-w0qj7mvsz5"]
-  --single                If specified, exit after running a single job.
+  --version                    Show version number                     [boolean]
+  --labels                     One or more user-defined labels for this runner
+                               (delimited with commas)          [default: "cml"]
+  --idle-timeout               Time in seconds for the runner to be waiting for
+                               jobs before shutting down. Setting it to 0
+                               disables automatic shutdown        [default: 300]
+  --name                       Name displayed in the repository once registered
+                                                     [default: "cml-{randomID}"]
+  --single                     Exit after running a single job
                                                       [boolean] [default: false]
-  --reuse                 If specified, don't spawn a new runner if there is a
-                          registed runner with the given labels.
+  --reuse                      Don't launch a new runner if an existing one has
+                               the same name or overlapping labels
                                                       [boolean] [default: false]
-  --driver                     If not specify it infers it from the ENV.
+  --driver                     Platform where the repository is hosted
```

Comment:
missing that can be inferred to some extend?

Reply:
Nice catch, @DavidGOrtega! 👀  I missed it when updating the comments.